instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -2519,3 +2529,165 @@ def get_anthropic_tool_config():
             },
         }
     ]
+
+
+@pytest.mark.skipif(
+    BOTO3_VERSION < (1, 34, 1), reason="InvokeAgent API not available"
+)
+@pytest.mark.vcr()
+def test_invoke_agent_with_completion(
+    span_exporter,
+    metric_reader,
+    bedrock_runtime_client,
+    bedrock_agent_client,
+    instrument_with_content,
+):
+    agent_id = "TFMZVIWXR7"
+    agent_alias_id = "ZT7YYHO8TO"
+    session_id = str(uuid.uuid4())
+    input_text = "What's the weather like in Seattle today?"
+
+    # Make the actual API call
+    response = bedrock_agent_client.invoke_agent(
+        agentId=agent_id,
+        agentAliasId=agent_alias_id,
+        sessionId=session_id,
+        inputText=input_text,
+        enableTrace=True,
+        endSession=False,
+    )
+
+    event_stream = response.get("completion")
```

Comment:
```suggestion
    event_stream = response["completion"]
```

You'll crash anyway if it's not there or not iterable

Reply:
done